lib/src/main/java/com/dynatrace/file/util/DynatraceFileBasedConfigurationProvider.java

```diff
@@ -73,11 +74,18 @@ private void setUp(String fileName, Duration pollInterval) {
 
   // This method should never be called by user code. It is only available for testing.
   // VisibleForTesting
-  void forceOverwriteConfig(String fileName, Duration pollInterval) {
+  public void forceOverwriteConfig(String fileName, Duration pollInterval) {
```

Comment:
Please use Javadoc here so it shows up properly in the IDE in case someone would ever try to use it.